src/components/eventLog/EventLogList.tsx

```diff
@@ -44,7 +45,11 @@ const EventLogList: React.FC<EventLogListProps> = ({
     <div data-cy="event-logs-table">
       <MaterialTable
         icons={tableIcons}
-        title={'Event logs'}
+        title={
+          <Typography variant="h6" component="h2">
+            {'Event logs'}
```

Comment:
I believe we can simply type `Event logs` instead of `{'Event logs'}` here because it is a string literal within a JSX